airbyte-integrations/connectors/source-sftp-bulk/source_sftp_bulk/client.py

```diff
@@ -196,10 +207,10 @@ def fetch_file(self, fn: Mapping[str, Any], file_type="csv") -> pd.DataFrame:
 
             raise Exception("Unable to read file: %s" % e) from e
 
-    def fetch_files(self, files, file_type="csv") -> Tuple[datetime, Dict[str, Any]]:
+    def fetch_files(self, files, separator, file_type="csv") -> Tuple[datetime, Dict[str, Any]]:
```

Comment:
Can you please add default value for the separator?
#Option 1 - Add "," as default separator value (as default file type is csv) 
#Option 2 - Add None as default separator value (but the existing pipelines which uses default values will incur Sniffer processing overhead.
So my suggestion is to use option 1 and add comments as passing None as separator will try to identify the separator using Sniffer.